ansible/roles/prometheus/templates/prometheus-ironic-exporter.conf.j2

```diff
@@ -0,0 +1,7 @@
+[oslo_messaging_notifications]
+# Template this config separately, as the ipmi exporter fails to parse the main ironic.conf
+# in the case that `driver` is defined twice, e.g. for messagingv2 AND prometheus_exporter.
+# Ironic exporter only requires the `location` option.
+driver = prometheus_exporter
+transport_url = fake://
```

Comment:
Does this work out of the box? The general advice is to have the defaults work for K-A things. As little additional operator config as possible.

Reply:
It does work out of the box, the exporter side has an extremely minimal configuration, and actually only depends on the line `location = /opt/stack/node_metrics`